charts/ExploreView.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react"
+import * as ReactDOM from "react-dom"
+
+import { Bounds } from "./Bounds"
+import { ChartView } from "./ChartView"
+import { ChartConfig } from "./ChartConfig"
+
+// Hardcoding some dummy config for now so we can display a chart.
+// There will eventually be a list of these, downloaded from a static JSON file.
+// -@jasoncrawford 2 Dec 2019
+const DUMMY_JSON_CONFIG = {
+    id: 677,
+    title: "Child mortality rate",
+    subtitle: "Share of newborns who die before reaching the age of five.",
+    sourceDesc: "IHME, Global Burden of Disease",
+    note: "",
+    dimensions: [{ display: {}, property: "y", variableId: 104402 }],
+    selectedData: [{ index: 0, entityId: 355 }]
+}
+
+// This component was modeled after ChartView.
+//
+// TODO that ChartView handles but this doesn't:
+// * re-render on window resize event (throttled)
+// * FullStory event logging on bootstrap
+// * error logging via Analytics.logEvent on componentDidCatch
```

Comment:
Is the intent eventually that this will replace `ChartView` or would it always be a wrapper?

Reply:
ExploreView will always contain a ChartView, I think